2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1357,4 +1355,32 @@ Sono classificati per priorità e per fonte.
 ==== Analisi dei requisiti
 Il documento _Analisi dei Requisiti_ raccoglie le informazioni previste. TODO contiene una descrizione approfondita del contenuto del documento. Esso include quanto descritto in questa sezione e contiene informazioni di tracciabilità. Il documento deve ricevere approvazione esplicita da parte degli stakeholder coinvolti.
 
-
+#pagebreak()
+
+= Tracciamento Paragrafi-ISO/IEC/IEEE 12207:2017
+
+La tabella di seguito riportata consente di associare ogni capitolo del documento al rispettivo capitolo dello standard di riferimento. Viene riporato anche il grado di conformità:
+- *to outcome* indica che il gruppo ha dovuto adattare lo standard al progetto, omettendo o reinterpretando sezioni incompatibili con la natura del progetto pur cercando il più possibile di perseguire l'obbiettivo di qualità che lo standard impone;
+- *full* indica che il capitolo riporta fedelmente le indacazioni dello standard con poche o nessuna azione di adeguamento.
+
+#figure(
+  table(
+    columns: 3,
+    fill: (col, row) => if row == 0 {rgb("#bbbbbb")},
+    [*Capitolo Norme*],[*Capitolo Standard*],[*Conformance Level*],
+    [@fornitura], align(left, "6.1.2 - Supply process"),[to outcome],
+    [@ciclo_di_vita], align(left, "6.2.1 - Life cycle model management process"),[to outcome],
```

Comment:
M maiuscola in Management

Reply:
management in questo caso risulta in minuscolo nello standard (tutti i nomi in realtà sono copia-incollati), procedo comunque con la modifica?

Reply:
Ok se nello std è così chiudo la convo